google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -574,6 +575,10 @@ def tables token: nil, max: nil
         #   passed is a hash `{ myparam: "foo" }`, the query must use named
         #   query parameters. When set, `legacy_sql` will automatically be set
         #   to false and `standard_sql` to true.
+        # @param [Hash<String|Symbol, External::Table>] external A Hash that
```

Comment:
Since this is a bit involved, a code example would be helpful if it's not too hard to add.

Reply:
Added code example.